data/data/openstack/topology/private-network.tf

```diff
@@ -62,6 +62,8 @@ resource "openstack_networking_port_v2" "masters" {
   allowed_address_pairs {
     ip_address = var.ingress_ip
   }
+
+  depends_on = [openstack_networking_port_v2.api_port, openstack_networking_port_v2.ingress_port]
```

Comment:
Since masters get random ips, we must add this dependency so that master are created after api and ingress ports.